Lykhoyda/add snap

Lykhoyda opened the pull request.

willemolding commented on packages/demo-wallet/dist/_headers:
This is used by Netlify (and other web hosting platforms) to set the headers correctly. Please put it back 😆

willemolding reviewed (approved):
If you we going to apply as global lint to the demo-wallet ( and we definitely should, I'm glad you added that) we should also include the .eslint config in that package so everyone's editors will respect the same style.

Other than that and the issue with removing the `_headers` file this looks good to merge

willemolding commented:
Please see https://github.com/Lykhoyda/WebZjs/pull/1/files

Lykhoyda commented:
> Please see https://github.com/Lykhoyda/WebZjs/pull/1/files

I think it's also a good strategy so far to make it more minimalistic as we don't require any customization since we split the wasm packages. I merged your PR and added some adjustments. Please take a look 👍
